### ISSUE TITLE
Redis batteries not inncluded in docker image

### ISSUE THREAD

**Issue by dudeskeeroo** [mautic/docker-mautic#252]
I'm attempting to use Redis as the queuing backend for Mautic 5. When I specify the redis settings in the queue configuration, it informs me that:

`The test message could not be sent due to the following error: 'Class "Redis" not found'.`

![image](https://github.com/mautic/docker-mautic/assets/16234856/5b0e790b-747f-41d1-828d-4d50df9e27f5)

The docker mautic build file doesn't contain the string "redis", but there are lib references in there for other schemes such as amqp.

Can Redis support be easily added to the mautic 5 docker image?

**Comment by mollux** [comment]
@dudeskeeroo 

that's correct, currently the Docker images contain the exact same dependencies as the Mautic project.  
Redis as messenger is not part of this, so this image has no out-of-the-box support for it.
This is something that can be discussed in [the main Mautic project](https://github.com/mautic/mautic)

Currently the way forward is to build a variant of the image that contains both the PHP redis extension and the composer dependency to be able to queue.

It makes to provide an example on how to do that, so if you can and want to contribute, please make a PR that extends the image with both the PHP extension as the messenger redis dependency

**Comment by IonutOjicaDE** [comment]
Hey there! I'm helping the new maintainers of this repository. It was neglected for some time but that changes now! Could you please re-test if you are still getting this issue so we could focus our limited time on the issues that are still relevant? If there will be no response we'll close this issue in 1 week. It can always be re-opened later on.

**Comment by dudeskeeroo** [comment]
I just pulled the latest 5.2.4 using `docker compose pull` and tried the `redis` scheme for email queue. It didn't work:

`The test message could not be sent due to the following error: 'Class "Redis" not found'.`

**Comment by cibero42** [comment]
Hello @dudeskeeroo,

Thaks for verifying that! As @mollux pointed out, Redis currently isn't supported, but we'll take this PR as a feature request, and keep you updated.

We're a small team, so it might take a long time until we'll come back to this issue. If you want to speed things up, you could create a PR to implement this :)
